rules/rules.md

```diff
@@ -62,20 +62,24 @@ handled. The faster robot can handle the slower robot as an obstacle, i.e.
 avoid it or wait until the obstacle disappears. In general the road rules
 will be respected: right of way, avoidance to the right, passing on the left.
 
-Autonomous and service area
+Autonomous, semiautonomous and service areas
 
 The competition area is described by polygon and it is called "Autonomous
 Area". The robots can move in this area in autonomous mode only. In the case of
 a collision, leaving the road or due to some other issues the contestants have
 to remove the robot from the Autonomous Area. The complementary space, called
 "Service Area", is the where all the maintenance and programming will take
-place.
+place. There is also "Semiautonomous Area" on the border of autonomous and
+service area. In semiautonomous area the robot has to navigate autonomously
+but in a case of problem the team can take the robot back to service area.
 
 Start
 
 All robots will start simultaneously on arbitrary park road within the service
 area. In the very minute of task announcement (position of loading and
-unloading area) the robots can start.
+unloading area) the robots can start. The coordinates will be defined via QR code.
+The entry in service area will be given by the contestants, but in loading and
```

Comment:
Contestants pass the entry to their robot in the service area, but organizers will enter the next destination in the loading and unloading zone.

(I.e. active voice instead of passive. English likes it more than Czech.)